scripts/docker.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# Build (and optionally upload to Dockerhub) for CLI $1 & version $2.
+# additionally --release, -r to release container to Dockerhub
+# note: if you want to release as well, you should be logged in with `docker`.
+
+set -e
+
+DIR=`dirname $0`
+ROOT=$DIR/..
+
+CLI=$1
+VSN=$2
+RELEASE=$3
+
+if [ -z "$CLI" ]; then
+    echo "require cli as first parameter" 2>&1
+    exit 1
+fi
+
+if [ -z "$VSN" ]; then
+    echo "require version as second parameter" 2>&1
```

Comment:
If the version is omitted, let's default to `latest` instead, as this is what docker defaults to when you don't specify a tag, e.g. in:

```bash
$ docker run ipinfo/cidr2ip 
Unable to find image 'ipinfo/cidr2ip:latest' locally
...
```